mygame.html

```diff
@@ -0,0 +1,87 @@
+<script>
+    var userNumber,
+      computerNumber = Math.floor(Math.random() * 10);
+
+
+  userNumber = prompt("Pick a number from 1 to 10!");
+
+    if (userNumber == computerNumber) {
+        message = "You get to drive a Ferarri!";
+    }
+    else{
+        message = "You get to drive a prius!";
+    };
+
+  alert(message);
+
+
+  userNumber = prompt("pick another number from 1 to 10!");
+
+  if (userNumber == computerNumber) {
+    message = "You're racing against a prius! This should be easy!";
+  }
+  else{
+    message = "Put the petal to the floor! You're up against a Ferarri!";
+  }
+  alert(message);
+
+
+  userNumber = prompt("Pick another number from 1 to 10 to see who gets off the line first!");
+
+  if (userNumber >= computerNumber) {
+    message = "You beat the other guy off the line! Hang on to your position!";
+  }
+  else{
+    message = "The other guy beat you off the line but there's still time to catch him!";
+  }
+  alert(message);
+  
+   
+  userNumber = prompt("Pick another number from 1 to 10 to try to use your NOS!");
+
+  if (userNumber >= computerNumber) {
```

Comment:
each one of these sections (prompt, if, alert) have the same structure. you could use a function instead, and call it with different strings.